packages/transition-backend/src/services/nodes/__tests__/NodeCollectionUtils.test.ts

```diff
@@ -132,3 +135,21 @@ test('saveAndUpdateAllNodes without collection manager', async() => {
     expect(savedObject4.getId()).toEqual(nodeAttributesFar.id);
     expect(savedObject4.getData('transferableNodes')).toEqual(nodeFarTransferableNodes);
 });
+
+test('saveAllNodesToCache without collection manager', async() => {
+
+    await saveAllNodesToCache(nodeCollection);
+
+    // Make sure all save calls to object to cache were done correctly
+    expect(mockedObjectToCache).toHaveBeenCalledTimes(4);
+    // Make sure nodes have been updated
+    const savedObject1 = mockedObjectToCache.mock.calls[0][0];
+    const savedObject2 = mockedObjectToCache.mock.calls[1][0];
+    const savedObject3 = mockedObjectToCache.mock.calls[2][0];
+    const savedObject4 = mockedObjectToCache.mock.calls[3][0];
+    // Validate node data was successfully saved
+    expect(savedObject1.getId()).toEqual(nodeAttributesClose1.id);
+    expect(savedObject2.getId()).toEqual(nodeAttributesClose2.id);
+    expect(savedObject3.getId()).toEqual(nodeAttributesClose3.id);
+    expect(savedObject4.getId()).toEqual(nodeAttributesFar.id);
```

Comment:
You can also check here that data.transferrableNode is correctly set with the mock mentioned above.